test/Chainweb/Test/Pact/Checkpointer.hs

```diff
@@ -369,6 +370,45 @@ checkpointerTest name initdata =
           runExec blockEnv09 Nothing "(m6.readTbl)" >>= \EvalResult{..} -> Right _erOutput @?= traverse toPactValue [tIntList [1,4]]
           _cpSave _cpeCheckpointer hash08
 
+          next "Don't create the same table twice in the same block"
+          hash09 <- BlockHash <$> merkleLogHash "0000000000000000000000000000009a"
+          blockEnv10 <- _cpRestore _cpeCheckpointer (Just (BlockHeight 9, hash08))
+
+          let h :: SomeException -> IO (Maybe String)
+              h = const (return Nothing)
+
+
+          let expectException act = do
```

Comment:
move these definitions to a `where` block

Reply:
done